### PR TITLE
decrese selenium/webdriver loglevel

### PR DESCRIPTION
**Pull request by mminder**
CI log is way to long, WARN should be sufficient

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by nmattia — COMMENTED** [review]
A case can definitely be made for having INFO logs when encountering a failure. Can we log everything to a file and add that as an artefact? And then only have WARN in the log output on CI?

**Comment by mminder** [comment]
> A case can definitely be made for having INFO logs when encountering a failure. Can we log everything to a file and add that as an artefact? And then only have WARN in the log output on CI?

Excellent idea. I just pushed a commit to try it out

**Comment by mminder** [comment]
@nmattia Looks good. I also added the screenshots as build artifact and also I'm taking another screenshot on error. Could be useful, what do you think?

File: src/frontend/src/test-e2e/util.ts [hunk 1]
```diff
@@ -57,6 +57,9 @@ export async function runInBrowserCommon(
     },
     automationProtocol: "webdriver",
     path: "/wd/hub",
+    logLevel: "info",
+    // outputDir pipes all webdriver log output into ./wdio.log
```
Comment (nmattia) [inline src/frontend/src/test-e2e/util.ts:61]:
```suggestion
    // outputDir pipes all webdriver log output into ./wdio.log
    // stdout only contains errors on test failures
```

**Review by nmattia — APPROVED** [review]
Looking great! One tiny suggestion.

File: src/frontend/src/test-e2e/util.ts (hunk 1 above)
Comment (mminder) [reply src/frontend/src/test-e2e/util.ts:61]: oh no, I just missed that comment! Will add it to a future PR

**Pull request merged**